hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/LocalIndexStatsImpl.java

```diff
@@ -21,47 +21,60 @@
 import com.hazelcast.json.internal.JsonSerializable;
 import com.hazelcast.query.LocalIndexStats;
 
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_AVERAGE_HIT_LATENCY;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_AVERAGE_HIT_SELECTIVITY;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_CREATION_TIME;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_HIT_COUNT;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_INSERT_COUNT;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_MEMORY_COST;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_QUERY_COUNT;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_REMOVE_COUNT;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_TOTAL_INSERT_LATENCY;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_TOTAL_REMOVE_LATENCY;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_TOTAL_UPDATE_LATENCY;
+import static com.hazelcast.internal.metrics.MetricDescriptorConstants.MAP_METRIC_INDEX_UPDATE_COUNT;
+
 /**
  * Implementation of local index stats that backs the stats exposed through the
  * public API.
  */
 @SuppressWarnings("checkstyle:methodcount")
 public class LocalIndexStatsImpl implements LocalIndexStats, JsonSerializable {
 
-    @Probe
+    @Probe(name = MAP_METRIC_INDEX_CREATION_TIME)
     private volatile long creationTime;
 
-    @Probe
+    @Probe(name = MAP_METRIC_INDEX_QUERY_COUNT)
     private volatile long queryCount;
 
-    @Probe
+    @Probe(name = MAP_METRIC_INDEX_HIT_COUNT)
     private volatile long hitCount;
 
-    @Probe
+    @Probe(name = MAP_METRIC_INDEX_AVERAGE_HIT_LATENCY)
     private volatile long averageHitLatency;
 
-    @Probe
+    @Probe(name = MAP_METRIC_INDEX_AVERAGE_HIT_SELECTIVITY)
```

Comment:
Does it make sense to mark this one with `PERCENT` or it's not applicable?

Reply:
Done